staging/src/k8s.io/apimachinery/pkg/fields/selector.go

```diff
@@ -33,6 +33,10 @@ type Selector interface {
 	// Empty returns true if this selector does not restrict the selection space.
 	Empty() bool
 
+	// Supported returns nil if this selector supports the given set of fields.
```

Comment:
It might be because of my english, but to be honest, I don't understand what that means.

What does it mean that "selector supports a given set of fields? That it contains only selections on fields from a given set? Or something different?